src/main/java/bio/terra/cloudfiletodatastore/deltalayer/model/json/PointCorrectionRequest.java

```diff
@@ -0,0 +1,68 @@
+package bio.terra.cloudfiletodatastore.deltalayer.model.json;
+
+import java.time.OffsetDateTime;
+import java.util.List;
+
+/**
+ * This class as well as the others in this package are used to deserialize delta layer json using
+ * Google's Gson library. files:
+ *
+ * <p>{ "destination": { "bqDataset": "point_correction", "datasetProject":
+ * "delta-layer-schema-tests", "workspaceId": "fe2dfaa7-e3a8-4725-b652-499f4eb2c88d" }, "insertId":
+ * "d3b7b01f-952e-4d5e-b4c2-548c21409da0", "insertTimestamp": "2021-07-23T14:23:33.060729Z",
+ * "inserts": [{ "datarepoRowId": "1f1d1041-bb84-4cc5-bb86-16fd40cc9a34", "name":
+ * "my_new_bool_column", "value": "false" }], "source": { "insertingUser": "111137720113699110757",
+ * "referenceId": "6d1f2766-f46c-4427-acc2-00caf089b37e" } }
+ */
+public class PointCorrectionRequest {
+
+  private String insertId;
+
+  private PointCorrectionDestination destination;
+
+  private List<PointCorrectionOperation> inserts;
+
+  private PointCorrectionSource source;
+
+  private OffsetDateTime insertTimestamp;
```

Comment:
same comment about `Instant` vs `OffsetDateTime`

Reply:
I think a zero offset OffsetDateTime is the same as an Instant.  I'm not sure sure there's harm here in using OffsetDateTime and there seems to be info suggesting it should be used when storing datetimes in BQ.  https://stackoverflow.com/questions/30234594/whats-the-difference-between-zoneddatetime-and-offsetdatetime